hugegraph-llm/src/hugegraph_llm/config/llm_config.py

```diff
@@ -33,13 +33,13 @@ class LLMConfig(BaseConfig):
     # 1. OpenAI settings
     openai_chat_api_base: Optional[str] = os.environ.get("OPENAI_BASE_URL", "https://api.openai.com/v1")
     openai_chat_api_key: Optional[str] = os.environ.get("OPENAI_API_KEY")
-    openai_chat_language_model: Optional[str] = "gpt-4o-mini"
+    openai_chat_language_model: Optional[str] = "gpt-4.1-mini"
```

Comment:
<img width="1246" alt="image" src="https://github.com/user-attachments/assets/fa9618bc-2da2-4b35-82c5-777447ffe83f" />

Ensure the key is compatible?